Makefile

```diff
@@ -5,7 +5,7 @@ all: poetry
 # --------
 
 PYTHON_BIN   := python3.10
-KEVM_PYK_DIR := ./kevm-pyk
+KEVM_PYK_DIR := $(abspath ./kevm-pyk)
```

Comment:
Why is this change needed? Also, why is `poetry.lock` updated? These changes should probably be backed out, should only be updated by the auto-updater.

Reply:
My version of Poetry (I think it is 1.6.1) complained about a relative path and this fixed it. I will try reverting `poetry.lock` and see whether it still works.